src/Core.UnitTests/CSharpVB/RuleSetGeneratorTests.cs

```diff
@@ -85,35 +86,35 @@ public void Generate_EmptyProperties()
         }
 
         [TestMethod]
-        public void Generate_ActiveRules_Always_Warning()
+        public void Generate_ActiveRules_VsSeverity_IsCorrectlyMapped()
         {
             // Arrange
             var generator = new RuleSetGenerator();
             
             var activeRules = new List<SonarQubeRule>
             {
-                CreateSonarCSharpRule("rule1", isActive: true),
-                CreateSonarCSharpRule("rule2", isActive: true),
+                CreateSonarCSharpRule("rule1", isActive: true, sqSeverity: SonarQubeIssueSeverity.Info),
+                CreateSonarCSharpRule("rule2", isActive: true, sqSeverity: SonarQubeIssueSeverity.Critical)
```

Comment:
This test doesn't check all of the possible severities - it's just checking that the severity mapping function is applied correctly. The severity mapping function is tested separately below.